docs/pages/database-access/guides/redshift-serverless.mdx

```diff
@@ -0,0 +1,400 @@
+
+<ScopedBlock scope={["oss", "enterprise"]}>
+![Teleport Database Access Redshift Self-Hosted](../../../img/database-access/guides/redshift_selfhosted_serverless.png)
+</ScopedBlock>
+<ScopedBlock scope={["cloud"]}>
+![Teleport Database Access Redshift Cloud](../../../img/database-access/guides/redshift_cloud_serverless.png)
+</ScopedBlock>
+
+## Prerequisites
+
+- A pre-existing Teleport `(=teleport.version=)` cluster. If you don't already
+  have Teleport running, see [How to Choose a Teleport Edition](../../choose-an-edition/introduction.mdx)
+  to learn more about your options, and how to set up Teleport.
+- AWS account with a Redshift Serverless configuration and permissions to create
+  and attach IAM policies.
+- Command-line client `psql` installed and added to your system's `PATH` environment variable.
+- A host where you will run the Teleport Database Service. This guide assumes an
+  EC2 instance, and provides a corresponding example of access control.
+- (!docs/pages/includes/user-client-prereqs.mdx!)
+
+## Step 1/5. Create an IAM policy for Teleport
+
+<Admonition type="note" title="Example Access Control">
+This guide provides an example configuration of IAM access roles as a model,
+and uses an EC2 instance to serve the Teleport Database Access service.
+The level of access provided may not suit your needs, or may not fit your
+organization's access conventions. You should adjust the AWS IAM permissions to
+fit your needs.
+</Admonition>
+
+Teleport needs AWS IAM permissions to be able to:
+
+- Discover and register Redshift Serverless workgroups and their associated VPC endpoints.
+- Manage IAM user or IAM role policies.
+
+If you don't already have one, create a role to apply to the EC2 instance that
+the Teleport Database Service will run on. This guide uses the example name
+`teleport-redshift-node`.
+
+Under **Trusted entity type** choose "AWS service", and under **Use case** choose "EC2".
+
+Create and attach a policy to the `teleport-redshift-node` role to allow Teleport
+to bootstrap access to AWS Redshift Serverless:
+
+```json
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Action": [
+                "iam:AttachRolePolicy",
+                "iam:PutRolePermissionsBoundary"
+            ],
+            "Resource": [
+                "arn:aws:iam::(=aws.aws_access_key=):role/teleport-redshift-node"
+            ],
+            "Effect": "Allow"
+        },
+        {
+            "Action": [
+                "iam:GetPolicy",
+                "iam:CreatePolicy",
+                "iam:TagPolicy",
+                "iam:ListPolicyVersions",
+                "iam:CreatePolicyVersion"
+            ],
+            "Resource": [
+                "*"
+            ],
+            "Effect": "Allow"
+        }
+    ]
+}
+```
+
+Remember to replace the example AWS account ID.
+
+<Admonition type="tip">
+This policy is required for Automatic Teleport bootstrapping, documented below.
```

Comment:
Are all parts of this policy required for bootstrapping? If so, would it make sense to move this text to the top of the section so readers don't reach this tip only after creating and attaching the policy?

Another possibility would be to move these instructions into the "Automatic" tab in the "Bootstrap access" section, since (as far as I understand it) they only apply to that tab

Reply:
Going with your first suggestion, since it makes sense to do all the AWS things at once, rather than go back into the AWS console when we're talking about running CLI commands on the node.